src/client/NFT.js

```diff
@@ -18,78 +18,36 @@ const {
  * @methodGroup Minting
  * @namedParams
  * @param {string} tenantId - The ID of the tenant
- * @param {string=} email - The email of the NFT recipient
  * @param {string=} address - The address of the NFT recipient
- * @param {string} collectionId - The ID of the NFT collection containing the NFT
- * @param {Array<Object>} items - List of items
- * @param {string} items.sku - SKU of the NFT
- * @param {number=} items.quantity=1 - Number to mint
- * @param {(string | Array<string>)=} items.tokenId - Custom token IDs for these items (must be unique).
- * @param {Object=} items.extraData={} - Additional data to put in the transaction
- * @param {Object=} extraData={} - Additional data to put in the transaction
+ * @param {string} marketplaceId - The ID of the marketplace containing the NFT
+ * @param {string} sku - SKU of the NFT in the marketplace
+ * @param {number=} quantity=1 - Number to mint
  *
  * @return Promise<Object> - An object containing the address for whom the NFT was minted and the transactionId of the minting request.
  */
-exports.MintNFT = async function({tenantId, email, address, collectionId, items, extraData={}}) {
```

Comment:
Does this API not support minting to an email address? I imagine Push relies on that feature, I'm not sure there's currently a way to ask for the user's address from the wallet, but they definitely have the email.

Reply:
I didn't think minting worked to an email address before.  I can't get it to work by just testing - I'll look at the authd code more carefully